lib/mixpanel-node.js

```diff
@@ -41,7 +41,7 @@ var create_client = function(token, config) {
         var event_data = new Buffer(JSON.stringify(data));
         var request_data = {
             'data': event_data.toString('base64'),
-            'ip': 0,
+            'ip': metrics.config.track_ip ? 1 : 0,
```

Comment:
I'd be fine with any of those, though I think "metrics.config.track_server_ip" is the most descriptive.  Though this represents a minority of cases, it might be more common than you  think.  For example, anyone doing development on electron would want this.